src/MosparoIntegration/Module/Account/AccountModule.php

```diff
@@ -13,15 +13,50 @@ public function __construct()
         $this->name = __('Account', 'mosparo-integration');
         $this->description = __('Protects the account forms like login, register and password reset with mosparo.', 'mosparo-integration');
         $this->dependencies = [];
+        $this->settings = [
+            'login_form' => [
+                'label' => __('Enable Wordpress login form protection', 'mosparo-integration'),
+                'type' => 'boolean',
+                'value' => true,
+            ],
+            'register_form' => [
+                'label' => __('Enable Wordpress registration form protection', 'mosparo-integration'),
+                'type' => 'boolean',
+                'value' => true,
+            ],
+            'lostpassword_form' => [
+                'label' => __('Enable Wordpress lost password form protection', 'mosparo-integration'),
+                'type' => 'boolean',
+                'value' => true,
+            ]
+        ];
```

Comment:
Maybe we could add a sentence (defined in the module like these settings) above the settings, like "Please choose which account forms you want to protect with mosparo." Then, we could shorten the labels for the settings to "WordPress login form" or "WordPress Registration Form."

Reply:
I reworked this part to include a ModuleSettings class (0a8a3c549e651919f87eaa76aa679b092a5c4c2d)  and allow more configuration on this part.
I added a 'form `header` parameter and changed texts to follow your suggestion.